### PR TITLE
Scaffolding

### LINKED ISSUE #3: Scaffolding
Begin file layout via branch "scaffolding".

### PR DESCRIPTION
**Pull request by LE0616**
If this looks good, I'd like to merge scaffolding, close the scaffolding issue, and move on to Feature: ContextMenus. See previous comment on this task for more.

### CODE REVIEW

File: sw.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+// Service Workers are replacing Background in Chrome MV3
+const bra = (greeting) => {
```
Comment (jkva) [inline sw.js]: Could potentially be simplified as

```javascript
const bra = greeting => `${greeting}, bra!`
```

as single-argument lambdas don't need parentheses and single-statement lambdas don't need explicit blocks.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import { bra } from '../sw'
+
+document.querySelector('h1').textContent = bra('Sup from SW.js');
```
Comment (jkva) [inline src/index.js]: Why is it called `bra`? I suspect this is trial code, but might make more sense to be called `greeting`?

**Review by jkva — CHANGES_REQUESTED** [review]
Does this proof-of-concept work as intended in Chrome? Other than that looks good. See comments.

File: sw.js (hunk 1 above)
Comment (LE0616) [reply sw.js]: It works as I intended but not as service-workers are intended at the moment. For now I'm just testing that the files can communicate with each other as expected. Full service-worker build and test is coming up.

File: src/index.js (hunk 2 above)
Comment (LE0616) [reply src/index.js]: yeah, its just funny trial code.  "greeting" works too :D

Comment (LE0616) [reply src/index.js]: Though I generally try not to name a function the same as it's input for variable clarity.

Comment (LE0616) [reply src/index.js]: How about greetingEdit?

**Review by jkva — APPROVED** [review]

**Comment by jkva** [comment]
@LE0616 feel free to merge and close. Good luck on the context menu 👍

**Comment by LE0616** [comment]
Delete the branch now that it's been merged or leave it for reference?

**Comment by jkva** [comment]
@LE0616 yeah, any merged branches can be removed as its work is captured in the target branch 👍

**Pull request merged**